tests/h/streamer/filter_test.py

```diff
@@ -218,11 +208,19 @@ def get_randomized_filter(self):  # pragma: no cover
                     "value": "3jgSANNlEeebpLMf36MACw" + str(random()),
                 },
                 {
-                    "field": "/references",
-                    "operator": "one_of",
+                    "field": "/group",
+                    "operator": "equals",
                     "value": [
-                        "3jgSANNlEeebpLMf36MACw" + str(random()),
-                        "3jgSANNlEeebpLMf36MACw" + str(random()),
+                        "333QnPxg" + str(random()),
+                        "333QnPxg" + str(random()),
+                    ],
+                },
+                {
+                    "field": "/user",
+                    "operator": "equals",
+                    "value": [
+                        "acct:@" + str(random()),
+                        "acct:@" + str(random()),
```

Comment:
I suggest formatting this as a valid userid (`acct:<non-empty username>@<authority>`).